src/config/configure.in

```diff
@@ -1141,13 +1142,17 @@ AS_HELP_STRING([--with-hip],
 
 AC_ARG_ENABLE(rocsparse,
 AS_HELP_STRING([--enable-rocsparse],
-               [Use rocSPARSE (default is YES).]),
+               [Use rocSPARSE (default is YES with HIP, otherwise NO).]),
 [case "${enableval}" in
     yes) hypre_using_rocsparse=yes ;;
     no)  hypre_using_rocsparse=no ;;
     *)   hypre_using_rocsparse=yes ;;
  esac],
-[hypre_using_rocsparse=yes]
+[case "$hypre_using_hip" in
```

Comment:
When `--enable-rocsparse=no` and `--with-hip=yes`, is `hypre_using_rocsparse` yes or no?

Reply:
the same q. for `cusparse`